tests/cmd/test_main.py

```diff
@@ -0,0 +1,78 @@
+import subprocess
+import unittest
+from hamcrest import equal_to, assert_that, is_, starts_with
+import socket
+import time
+import sys
+
+import os
+
+TEST_BIND_HOST = '127.0.0.1'
+TEST_BIND_PORT = '17333'
+
+
+class FakeSwitchesTest(unittest.TestCase):
+    def test_fake_switches_entrypoint_help(self):
+        p = subprocess.check_output(get_base_args() + ['-h'])
+
+        assert_that(p, is_("""fake_switches inline help
```

Comment:
If the main had a constant with the help you could assert it here instead of rewriting it